Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet.UWP/App.xaml.cs

```diff
@@ -19,6 +19,7 @@ sealed partial class App : Application
         /// </summary>
         public App()
         {
+            MobileCenter.SetCountry("United States");
```

Comment:
I feel uncomfortable that we reuse the SIM carrier country field for another purpose.
Shouldn't we change schema to introduce a new field?